src/main/java/org/example/commerce_site/application/product/ProductService.java

```diff
@@ -15,7 +18,26 @@
 public class ProductService {
 	private final ProductRepository productRepository;
 
+	@Transactional
 	public Product create(ProductRequestDto.Create productRequest, Category category) {
 		return productRepository.save(ProductRequestDto.Create.toEntity(productRequest, category));
 	}
+
+	@Transactional(readOnly = true)
+	public Product getProduct(Long productId) {
+		return productRepository.findById(productId).orElseThrow(
+			() -> new CustomException(ErrorCode.PRODUCT_NOT_FOUND)
+		);
+	}
+
+	@Transactional
+	public Product update(Product product, ProductRequestDto.Put dto, Category category) {
+		product.update(dto, category);
+		return productRepository.save(product);
+	}
+
+	@Transactional
+	public void 	delete(Product product) {
```

Comment:
```suggestion
	public void delete(Product product) {
```
이런 단순한 포맷팅 실수는 일어나기 쉬운 반면, 코드 가독성은 떨어뜨립니다. 팀 단위로 작업하면 이런 실수는 더더욱 자주 일어날 텐데, 어떻게 처리해볼 수 있을까요?

Reply:
naver-intellij-formatter.xml 를 적용했는데, 커밋 전마다 잊지말고 반드시 코드 포맷팅을 실행하기...?👀

Reply:
github action 등을 활용해서 CI로 포맷팅을 검사하거나, git commit 할 때 포매팅이 돌도록 실행해보는 방법이 있을 것 같습니다

Reply:
관련해서 찾아봤는데 인텔리제이에 commit 시 자동으로 Reformat code  와 Optimize imports 해주는 옵션이 있어 이를 사용하려고 합니다